handlers/restart.yml

```diff
@@ -0,0 +1,21 @@
+# file: postgresql/handlers/restart.yml
+  
+#
+# Restart PostgreSQL
+#
+- name: PostgreSQL | Force a checkpoint before a restart
```

Comment:
Doing a checkpoint is best-practice, as it reduces the actual downtime during the restart.